FIAT/guzman_neilan.py

```diff
@@ -11,17 +11,30 @@
 
 from FIAT import finite_element, polynomial_set, expansions
 from FIAT.bernardi_raugel import ExtendedBernardiRaugelSpace, BernardiRaugelDualSet, BernardiRaugel
+from FIAT.alfeld_sorokina import AlfeldSorokina
 from FIAT.brezzi_douglas_marini import BrezziDouglasMarini
 from FIAT.macro import AlfeldSplit
 from FIAT.quadrature_schemes import create_quadrature
+from FIAT.restricted import RestrictedElement
+from FIAT.nodal_enriched import NodalEnrichedElement
 from itertools import chain
 
 import numpy
 import math
 
 
-def ExtendedGuzmanNeilanSpace(ref_el, subdegree, reduced=False):
-    """Return a basis for the extended Guzman-Neilan space."""
+def ExtendedGuzmanNeilanSpace(ref_el, order, kind=1, reduced=False):
```

Comment:
Guzman-Neilan or extended?  Name and doc string don't match.

Reply:
I got rid of `Extended` in all classes and methods, and confined it to docstrings.